script.js

```diff
@@ -0,0 +1,38 @@
+let btn = document.getElementById('open-modal');
+let modal = document.querySelector('.modal')
+let modal_close = document.querySelector('.modal__close')
+let close_modal = document.getElementById('btn-close-modal')
+let modalBody = document.querySelector('.modal__body')
+let openSnackbarSuccess = document.getElementById('open-snackbar-success')
+let openSnackbarSecondary = document.getElementById('open-snackbar-secondary')
+let snackbarPrimary = document.querySelector('.snackbar__primary')
+let snackbarSecondary = document.querySelector('.snackbar__secondary')
+let closeSnackbarPrimary = document.querySelector('.close__snackbar__primary');
+let closeSnackbarSecondary = document.querySelector('.close__snackbar__secondary');
+
+
+btn.addEventListener('click',() => {
+    modal.style.display = 'block';
+    
+})
+modal_close.addEventListener('click',() =>{
```

Comment:
I would suggest not to use underscore naming conventions while writing JS as you are already using camelCase :)

Reply:
Sure I will follow it and scale during the course